tests/installation/addon_products_sle.pm

```diff
@@ -162,16 +162,13 @@ sub run {
     # Trap the 'missing license file on media' issue
     # Needle is configured as a workaround, so a soft-fail will be shown
     send_key 'alt-s' if check_screen('license-insert-disc-issue', 30);
-    # Full_installer jumps to the Select extension and modules dialog and this
-    # makes the tests fail as it doesnt find anything to match
-    unless ((check_var('FLAVOR', 'Full')) || ((is_sle('15-SP2+') && get_var('MEDIA_UPGRADE')))) {
-        if ($self->process_unsigned_files([qw(inst-addon addon-products)])) {
-            assert_screen_with_soft_timeout(
-                [qw(inst-addon addon-products)],
-                timeout      => 60,
-                soft_timeout => 30,
-                bugref       => 'bsc#1123963');
-        }
+    # Wait for the addon products screen
+    if ($self->process_unsigned_files([qw(inst-addon addon-products)])) {
```

Comment:
I guess we should still keep condition for the upgrade, as it's not related, or you think differently?

Reply:
I've triggered https://openqa.suse.de/tests/3765174 to see if change is safe for the migration scenarios.

Reply:
After a quick test this condition is still needed. It could works without it but the same side effect I saw: on slow HW/worker we can have issue because the needle doesn't appear in this case.